converter/include/k4SimDelphes/DelphesEDM4HepConverter.h

```diff
@@ -105,6 +110,7 @@ namespace k4SimDelphes {
 
     void processParticles(const TClonesArray* delphesCollection, std::string const& branch);
     void processTracks(const TClonesArray* delphesCollection, std::string const& branch);
+    void processCandidates(const TClonesArray* delphesCollection, std::string const& branch);
```

Comment:
```suggestion
    void processPFlowCandidates(const TClonesArray* delphesCollection, std::string const& branch);
```
or another slightly more descriptive name than just `Candidates`.